docs/usage/cli_package.md

```diff
@@ -26,5 +27,7 @@ optional arguments:
                         changes made by qgis-plugin-ci.
   --disable-submodule-update
                         If omitted, a git submodule is updated. If specified, git submodules will not be updated/initialized before packaging.
+  --auto-experimental
+                        If omitted, the flag experimental will be determined by the GitHub release. If set, the flag will be set according to the tag using https://semver.org
```

Comment:
can you explain what kind of tag will be considered as experimental?

Reply:
and/or give examples

Reply:
I think it's explained by the semver convention: everything with a beta, alpha, rc, etc.